katago_server/games/managers/rating_game_pandas_queryset.py

```diff
@@ -15,6 +15,7 @@ def get_total_games_count_as_white(self, run: Run):
         return (
             self.filter(run=run)
             .values(reference_network=F("white_network__pk"), opponent_network=F("black_network__pk"),)
+            .order_by()
```

Comment:
That's a mystery. 
👻

Reply:
I'd like to understand why, at some point though...